### DISCUSSION TITLE
Comment on website images

### DISCUSSION THREAD

**Discussion by WhyNotHugo** [mustang-im/mustang#336] [General]
When I visit https://parula.beonex.com/, there is a huge 1280x578 image of a landscape with a river across it.

If I scroll further, there is a tiny 455x286px screenshot of the email application. The screenshot is centred, but less than 25% of my view-port.

I suggest inverting priorities here: the landscape is very beautiful, but provides no context. The screenshot would provide context, but is too small to discern anything.

**Comment by WhyNotHugo** [comment]
I see that this repository now points to https://www.mustang.im/

The [upcoming FOSDEM talk](https://fosdem.org/2025/schedule/event/fosdem-2025-4887-parula-presenting-the-new-email-client/) points to https://parula.beonex.com

You probably want to update one of these.

**Reply by benbucksch** [reply]
The duality of Mustang vs. Parula is intentional for now until certain questions are resolved. Parula is the public name, Mustang is the internal code name.

**Comment by benbucksch** [comment]
@elenaux Elena?
